exercises/reverse-string/README.md

```diff
@@ -0,0 +1,106 @@
+# Reverse String
+
+The introductory exercise to arrays and strings. Just reverse input string.
+
+This is a typical first program for beginning programming in a new language
+or environment after being able to do simple tasks.
+
+The objectives are simple:
+
+- Write a function that reverses the input string.
+- Run the test suite and make sure that it succeeds.
+- Submit your solution and check it at the website.
+
+If everything goes well, you will be ready to fetch your first real exercise.
+
+## Tutorial
+
+This exercise has two files:
```

Comment:
This is to specific for a generic exercise description. My Erlang exercises won't have those 2 files, but a lot of other files.

Reply:
I agree. The tutorial shouldn't really be here, as it is JS specific.